avocado/core/test.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, uid, name, variant=None, no_digits=None):
             self.str_uid = str(uid).zfill(no_digits if no_digits else 3)
         else:
             self.str_uid = str(uid)
-        self.name = name or "<unknown>"
+        self.name = name
```

Comment:
Before the change:
```
>>> test.TestName(0, None)
>>> '0-<unknown>'
```
After the change:
```
>>> test.TestName(0, None)
>>> '0-None'
```
You're basically exchanging a human-friendly string for a machine readable value converted to string, but nothing really changes from the abuse-protection perspective. How exactly do you plan to (hopefully) flag abuses? :)